hs/ConvertSVGToPNG.hs

```diff
@@ -0,0 +1,48 @@
+module ConvertSVGToPNG where
+
+import System.Process
+import GHC.IO.Handle.Types
+import Control.Monad.IO.Class  (liftIO)
+import GHC.IO.Exception
+
+-- | Converts an SVG file to a PNG file. Note that image magik's 'convert' command
+-- can take in file descriptors.
+convertSVGToPNG :: String -> String -> IO
+                     (Maybe Handle,
+                      Maybe Handle,
+                      Maybe Handle,
+                      ProcessHandle)
+
+convertSVGToPNG inName outName = createProcess $ CreateProcess
+                                  (ShellCommand $ "convert ../res/graphs/graph_regions.svg INSERT_ID-graph.png"
+                                   )
+                                  Nothing
+                                  Nothing
+                                  CreatePipe
+                                  CreatePipe
+                                  CreatePipe
+                                  False
+                                  False
```

Comment:
I think we should generally keep as up to date as possible, until we use cabal to specify certain versions of dependencies.